e2e/commands/formOperations.ts

```diff
@@ -1,4 +1,59 @@
 import { APIRequestContext } from "@playwright/test";
+import customSchema from "../support/customSchema.json";
+
+export const createForm = async (api: APIRequestContext) => {
+  const formResponse = await api.post("form", {
+    data: {
+      name: "Form created for testing",
+      version: "1.0",
+      published: false,
+      description: "This is the form description",
+      encounterType: {
+        uuid: "e22e39fd-7db2-45e7-80f1-60fa0d5a4378",
+      },
+    },
+  });
+  const form = await formResponse.json();
+
+  return form;
+};
+
+export const addFormResources = async (
+  api: APIRequestContext,
+  valueReference: string,
+  formUuid: string
+) => {
+  await api.post(`form/${formUuid}/resource`, {
+    data: {
+      name: "JSON schema",
+      dataType: "AmpathJsonSchema",
+      valueReference: valueReference,
+    },
+  });
+};
+
+export const createValueReference = async (api: APIRequestContext) => {
+  const boundary =
+    "--------------------------" + Math.floor(Math.random() * 1e16);
+  const delimiter = "\r\n--" + boundary + "\r\n";
+  const closeDelimiter = "\r\n--" + boundary + "--";
+
+  const body =
+    delimiter +
+    'Content-Disposition: form-data; name="file"; filename="schema.json"\r\n' +
+    "Content-Type: application/json\r\n\r\n" +
+    JSON.stringify(customSchema) +
+    closeDelimiter;
```

Comment:
Will this help? https://developer.mozilla.org/en-US/docs/Web/API/FormData/Using_FormData_Objects

Reply:
The issue is `new FormData();` , `FormData` object is getting undefined. That's why this approach was followed in the first place.

Reply:
Could you please elaborate more on this?

Reply:
```
export const createValueReference = async (api: APIRequestContext) => {
  const schemaBlob = new Blob([JSON.stringify(customSchema)], {
    type: undefined,
  });
  const body = new FormData();
  body.append("file", schemaBlob);

  const valueReference = await api.post("clobdata", {
    data: body,
    headers: {
      "Content-Type": "multipart/form-data",
    },
  });

  return await valueReference.text();
};
```

<img width="921" alt="image" src="https://github.com/openmrs/openmrs-esm-form-builder/assets/43912578/406f43d7-726b-4236-8407-6de3c1ace5ca">


Same goes for form data as well.

Reply:
Form data is undefined. Tried importing as well.

Reply:
<img width="707" alt="image" src="https://github.com/openmrs/openmrs-esm-form-builder/assets/33048395/b58a5a12-8d78-497f-a5fc-e4e0b5ac1aaa">
<img width="707" alt="image" src="https://github.com/openmrs/openmrs-esm-form-builder/assets/33048395/8c577615-ec5c-4c26-a060-bb60937e2304">

Reply:
You don't really import them.

Reply:
can you try the exact code snippet? maybe checkout to my branch?